catalogue/webapp/components/WorkDetails/WorkDetails.tsx

```diff
@@ -343,7 +343,7 @@ const WorkDetails: FunctionComponent<Props> = ({
 
                 {/* Note: there is no class flex-h-center, but there is flex--h-center
                     Is that what's meant here? */}
-                <div className="flex flex-h-center">
+                <div className="flex-h-center" style={{ display: 'flex' }}>
```

Comment:
I just tested this with `flex--h-center` and I think this wasn't meant to be there and could be fully removed.

I don't think we want this:
<img width="659" alt="Screenshot 2023-04-11 at 09 36 25" src="https://user-images.githubusercontent.com/110461050/231104298-8a726440-ac7f-47e9-98d3-b4f44cebe4d5.png">